src/RegistryPageController.php

```diff
@@ -283,13 +350,15 @@ public function show($request)
             return $this->httpError(404);
         }
 
-        $data = DataObject::get_by_id($this->DataClass, $request->param('ID'));
+        $entry = DataObject::get_by_id($this->DataClass, $request->param('ID'));
 
-        if (!($data && $data->exists())) {
+        if (!($entry && $entry->exists())) {
```

Comment:
Existing code I know, but do you think that `if (!$entry || !$entry->exists()) {` might be slightly easier to read?